README.md

```diff
@@ -49,4 +49,4 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-```
\ No newline at end of file
+```
```

Comment:
Yep, It's true, thanks, but please do not pollute your commits by modifying blank lines. Redo your patch without altering line 52 please.